engine/schema/src/main/java/com/cloud/dc/dao/ClusterDaoImpl.java

```diff
@@ -81,7 +81,6 @@ public ClusterDaoImpl() {
 
         ZoneSearch = createSearchBuilder();
         ZoneSearch.and("dataCenterId", ZoneSearch.entity().getDataCenterId(), SearchCriteria.Op.EQ);
-        ZoneSearch.groupBy(ZoneSearch.entity().getHypervisorType());
```

Comment:
@slavkap check if this can cause regression, for listing hypervisors for zone.

Reply:
@sureshanaparti I have tested it. My setup is 2 KVM cluster with 2 hosts on each. Before the change when listing hypervisors:

> list hypervisors zoneid=307c2b69-86eb-451e-8517-6113c59648c4
> { "count": 1,  "hypervisor": [ { "name": "KVM" } ] }

with the change when listing hypervisors:

> list hypervisors zoneid=307c2b69-86eb-451e-8517-6113c59648c4
> { "count": 2,  "hypervisor": [  { "name": "KVM"  },   {  "name": "KVM"  } ] }

Reply:
@slavkap list hypervisors should return only 1 KVM.

Reply:
ok than, I wasn't sure that it should return only 1 KVM if we have for example 3 clusters of them. I will implement another method, which gets all clusters in the zone. It's needed for StatsCollector to list all clusters in the zone, otherwise we get only one cluster of a hypervisor type.